pkgs/misc/drivers/sundtek/default.nix

```diff
@@ -10,18 +10,19 @@ let
     else
     if isi686 then "32bit"
     else throw "${system} not considered in build derivation. Might still be supported.";
+  sha256 = with stdenv;
+    if isx86_64 then "1jfsng5n3phw5rqpkid9m5j7m7zgj5bifh7swvba7f97y6imdaax"
+    else "15y6r5w306pcq4g1rn9f7vf70f3a7qhq237ngaf0wxh2nr0aamxp";
 
 in
   stdenv.mkDerivation {
     src = fetchurl {
       url = "http://www.sundtek.de/media/netinst/${platform}/installer.tar.gz";
-      sha256 = "15y6r5w306pcq4g1rn9f7vf70f3a7qhq237ngaf0wxh2nr0aamxp";
+      sha256 = sha256;
```

Comment:
can someone tell me how this worked before?

Reply:
Only on one platform I guess 😂